include/angle.h

```diff
@@ -6,20 +6,27 @@
 /*   By: ycarro <ycarro@student.42.fr>              +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2023/02/22 13:57:17 by ycarro            #+#    #+#             */
-/*   Updated: 2023/02/22 14:05:01 by ycarro           ###   ########.fr       */
+/*   Updated: 2023/02/23 14:19:58 by ycarro           ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #ifndef ANGLE_H
 # define ANGLE_H
 
-# include "libft.h"
-# include "error_code.h"
+# include <math.h>
+
+# include "player.h"
 # include "direction.h"
 
 typedef struct s_angle
 {
 	double	radians;
+	double	min_in_range;
+	double	max_in_range;
 }				t_angle;
 
-#endif
+void	starting_angle(t_angle *angle, t_player *player);
```

Comment:
it is better to have a common function prefix for each class
starting_angle -> angle_starting
set_angle_range -> angle_set_range